css/main.css

```diff
@@ -292,9 +292,10 @@ span.close:hover{
 }
 .resItem {
 	height: 30%;
+    width: 100%;
 	overflow-x: auto;
 	overflow-y: hidden;
-	margin-right: 1%;
+	margin: 0% 1%;
```

Comment:
You don't need the percent when you use 0 - it's considered bad form to put anything (px or %) with a 0 in css.

Reply:
good to know, thanks